### PR TITLE
Pin Runelite plugin by default

### LINKED ISSUE #6256: Pin RuneLite "plugin" by default
This is to help people who start using the client find the general RuneLite settings more easily. As the list is quite long and a lot of people tend to think RuneLite settings is referring to the plugin sidebar and not to the RuneLite config entry in the list.

### PR DESCRIPTION
**Pull request by beaumitch**
Resolves #6256

noConfig

This only effects new users without a config CSV.

### CODE REVIEW

**Comment by raiyni** [comment]
this does nothing for most users though

File: runelite-client/src/main/java/net/runelite/client/plugins/config/ConfigPanel.java [hunk 1]
```diff
@@ -650,7 +650,7 @@ void stopPlugin(Plugin plugin, PluginListItem listItem)
 
 		if (config == null)
```
Comment (Adam-) [inline runelite-client/src/main/java/net/runelite/client/plugins/config/ConfigPanel.java]: What happens when all plugins are unpinned? I think this becomes an empty string.

**Comment by beaumitch** [comment]
> 
> this does nothing for most users though

To effect existing users I think we'd have to add an ugly key to the config specifically for overriding their current setting to avoid overwriting the setting on each config reload. This at least effects new users - which are the most likely to need this feature.

**Comment by Adam-** [comment]
I wouldn't mind seeing the RL config just permanently pinned at the top. It's always hard for users to find.

**Comment by beaumitch** [comment]
> 
> 
> I wouldn't mind seeing the RL config just permanently pinned at the top. It's always hard for users to find.

Since space isn't really an issue in that panel, I'm leaning in this direction, too.

**Comment by deathbeam** [comment]
I said it multiple times before, but that is not because its not pinned at top, but because it is named "RuneLite". So simple solution is to rename it to something that makes sense when someone asks.

**Comment by deathbeam** [comment]
> > I wouldn't mind seeing the RL config just permanently pinned at the top. It's always hard for users to find.
> 
> Since space isn't really an issue in that panel, I'm leaning in this direction, too.

I disagree, if we want to "pin" it at top, it should be possible to unpin it. "RuneLite" do not contains any settings you would want to change often, and the pin feature was made for that wasnt it, so it will be taking up useless space at top.

**Comment by beaumitch** [comment]
> 
> 
> > > I wouldn't mind seeing the RL config just permanently pinned at the top. It's always hard for users to find.
> > 
> > 
> > Since space isn't really an issue in that panel, I'm leaning in this direction, too.
> 
> I disagree, if we want to "pin" it at top, it should be possible to unpin it. "RuneLite" do not contains any settings you would want to change often, and the pin feature was made for that wasnt it, so it will be taking up useless space at top.

This is a good point, potentially "Client" or something like that. No longer interested in seeing this merged, so I'll close for now. Tagging #6256 to preserve conversation.

**Pull request closed without merging**